package.json

```diff
@@ -35,6 +35,7 @@
     "@slack/events-api": "^2.1.0",
     "ajv": "6.5.3",
     "autoprefixer": "^9.1.5",
+    "axios": "^0.18.0",
```

Comment:
Tem o `make-requests` também. precisamos ver para escolher um (aposto que o axios vence) e manter um só no projeto, né?

Reply:
Precisa  arrumar o readme do teu make-requests, quando inseri o axios, pequisei antes e testei uns 3 packages, inclusive o default. Não tinha visto o make-requests. Se for simples de usar podemos continuar com o make-requests, não vejo problema. Arruma o readme com a instruções de uso, publicitar seu repositório, pede ajuda ao @rtarantelli pro README.md =:D

Reply:
Faço isso hoje! =) vamos seguir no axios aí mesmo pra não te prender esperando o readme, bele bele?